projects/batfish-common-protocol/test/src/org/batfish/test/TestSerialization.java

```diff
@@ -1,23 +0,0 @@
-package org.batfish.test;
-
-import org.batfish.datamodel.BgpNeighbor;
-import org.batfish.datamodel.Prefix;
-import org.codehaus.jettison.json.JSONException;
-import org.codehaus.jettison.json.JSONObject;
-import org.junit.Test;
-
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-
-public class TestSerialization {
-
-   @Test
-   public void test() throws JsonProcessingException, JSONException {
-      ObjectMapper mapper = new ObjectMapper();
-      BgpNeighbor neighbor = new BgpNeighbor((Prefix) null, null);
-      String val = mapper.writeValueAsString(neighbor);
-      JSONObject obj = new JSONObject(val);
-      assert obj.length() == 1; // just id
```

Comment:
comment: the actual size is 25. The test fails in IntelliJ because it turns on java assertions. The test passes with `ant test` because the `assert` is a no-op at runtime. If I change it to use more standard junit assertThat:

```java
assertThat(obj.length(), equalTo(1));
```

then the test fails because 1 != 25.